edx_sga/sga.py

```diff
@@ -70,7 +70,7 @@ class StaffGradedAssignmentXBlock(XBlock):
         scope=Scope.settings
     )
 
-    points = Float(
+    points = Integer(
         display_name="Maximum score",
         help=("Maximum grade score given to assignment by staff."),
         values={"min": 0, "step": .1},
```

Comment:
This "step" looks a bit suspicious for an Int. I did try changing it to 1. The problem is it still allows you to enter a float for the field when editing in Studio and then when saving you get an error and then I can't reload the unit. I wish I knew how to restrict input in the field (in Studio edit) to an int.